SUPPORT.md

```diff
@@ -0,0 +1,11 @@
+We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using InSpec), we recommend that if you are a:
```

Comment:
```suggestion
We use GitHub issues to track bugs and feature requests. If you need support help (i.e. with using Chef InSpec), we recommend that if you are a:
```

Reply:
Went ahead and got this added to the latest commit